convert/convert.go

```diff
@@ -98,6 +98,7 @@ func convertFile(path string, overwrite bool, importPath string, protocPath stri
 
 	result, err := format.Source(b.Bytes())
 	if err != nil {
+		fmt.Println(b.String())
```

Comment:
You missed this one I guess :)

Reply:
This was actually on purpose. Many things I tried added syntax errors, so format.Source would say "invalid syntax at line 6 ...".

So I think it's reasonable for convert to output the generated code when erroring like this. Not ideal, but at least better than giving an obscure error with no more info.

Reply:
OK! maybe the gunk/log instead ? Or a comment ? 
I'm afraid that this could be removed at some point because it's `fmt.Println`

Reply:
Done - it's now a Fprintln with a comment.